front/pages/w/[wId]/data-sources/vaults/[vaultId]/categories/[category]/index.tsx

```diff
@@ -66,6 +69,47 @@ export const getServerSideProps = withDefaultUserAuthRequirements<
   };
 });
 
+const fetchWebsiteRootUrl = async (
```

Comment:
We do a try catch and sometimes we throw to go in the catch and sometimes we return null, could we harmonize? 
We could remove the try/catch, return null whenever we can't retrieve the valid route?